interactions/api/models/user.py

```diff
@@ -63,3 +65,20 @@ def mention(self) -> str:
         :rtype: str
         """
         return f"<@{self.id}>"
+
+    def avatar_url(self, animated: Optional[bool] = True) -> str:
+        """
+        Returns the URL of the user's avatar
+
+        :param animated: If available, return the animated version of the avatar.
+        :type animated: Optional[bool]
```

Comment:
It *works*, but what's the usecase for adding an animated bool here versus just checking the avatar attribute itself?

Reply:
Checking only the attribute could make this a property couldn't it?

Reply:
That's what I'm suggesting;
You can check if self.avatar exists and if it starts with `a_` and making it a property, which makes it less stressful for bot devs to think about implementing an animated arg, albeit a minor detail

Reply:
> It _works_, but what's the usecase for adding an animated bool here versus just checking the avatar attribute itself?

Maybe if a user only wants a picture and never a GIF, Idk. I just added it if it could be useful but I'll remove it and make it always return a GIF if the avatar is animated.